modules/ROOT/pages/migration-from-5x.adoc

```diff
@@ -3,88 +3,585 @@
+
+=== tinymce.activeEditor.options.get
+
+Previously `tinymce.activeEditor.options.get('plugins')` returned the configured plugins as a string.
+
+As of {productname} 6, `tinymce.activeEditor.options.get('plugins')` now returns the configured plugins as an array.
+
+
+[[plugins]]
+== Plugins
+
+=== Plugin loading format change
+
+{productname} 6.0 allows plugin loading in any of three formats:
 
-For information on using the latest development and testing builds, see: xref:editor-plugin-version.adoc[Cloud deployment guide - Specify editor & plugin versions].
+. An array of strings:
++
+`plugins: [ 'advlist', 'list', 'image', 'help', 'wordcount' ]`
++
+. A space-separated string:
++
+`plugins: 'advlist list image help wordcount'`
++
+. A comma-separated string:
++
+`plugins: 'advlist,list,image,help,wordcount'`
 
-[[plugin-changes]]
-== Plugin Changes
+Previously, two other formats were allowed: comma-separated strings with spaces and arrays of space-separated strings.
 
-=== Tabfocus plugin removed
+Neither of these formats are allowed in {productname} 6.0.
 
-The Tabfocus plugin has been removed in {productname} 6.0. This includes the `+tabfocus_elements+` option.
+=== Removed plugins
 
-Instead, tab order should be configured by specifying the https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex[`+tabindex+` attribute] on the target element for the editor, or the equivalent if using a framework. This allows the tab order to be handled by the browser. In inline mode, this works as in {productname} 5.x. In iframe (classic editor) mode, {productname} now copies the `+tabindex+` attribute from the target element to the editor's iframe, to allow this to work correctly.
+All references to any of these plugins should be removed from plugin configurations.
 
-This allows integrators to define the tab order of the page using standard HTML attributes, without needing to specially configure {productname}.
+NOTE: Leaving references to removed plugins in a plugin configuration will cause console errors documenting the plugin failing to load. {productname} 6.0 should, otherwise, work without error. If, however, {productname} 6.0 is ‘installed’ incorrectly — by being unZipped onto an existing {productname} 5.0 instance for example — the old plugin will not be removed, it will load and, since a {productname} 5.x plugin is attempting to work with a {productname} 6.0 instance, error notifications will almost certainly present to the end-user.
 
-For information on specifying tab order, see: xref:accessibility.adoc#taborder[Tab order].
+See below for specific documentation of potential migration issues, if any, regarding these removed plugins.
 
-[[option-setting-changes]]
-=== Option/Setting changes
+Seven `plugins` were removed because their functionality is now a Core part of {productname} 6.0:
 
-[[renamed-options]]
-==== Renamed Options
+. `colorpicker`
+. `contextmenu`
+. `hr`
+. `noneditable`
+. `paste`
+. `print`
+. `textcolor`
```

Comment:
```suggestion
. `textcolor`

The functionality from these plugins can now be used without referencing them in the plugin configuration.
```
Perhaps? When I was reading this the first time, something about this list bothered me, regarding clarity of this list vs. the next. I think this helps?